core/domain/entities/routing/handlers/admin/AdminRoute.php

```diff
@@ -53,6 +65,21 @@ public function __construct(
     }
 
 
+    public function getCapCheck()
+    {
+        return new CapCheck('edit_posts', 'access Event Espresso admin route');
```

Comment:
default cap check for admin routes

Reply:
right, i basically wanted a cap that would be typical of any role higher than subscriber.
the idea is that this cap check would be a fallback if nothing else is set for a route, so it's intentionally trying to allow nearly everyone through.